css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
+  position:absolute;
+  top: 8rem;
+  padding: 120px 0 0 150px;
+  height: 32em;
+  right: -20em;
+  gap: px;
+}
+
+.main-text{
+    width: 50%;
+    grid-area: right;
+    margin: auto;
+    position: absolute;
+    margin-right: 0px;
+    height: 510px;
+  
+}
+.name-title {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+
+}
+.name{
+    margin-top: 20px;
+    height: 20px;
+    font-family: sans-serif;
+    margin-left: 30px;
+}
+.name a{ 
+    line-height: 0.5em;
+    font-size: 20px;
+    margin-left: -20px;
+    
+}
+.title p{
+    font-size: 20px;
+    margin-left: 10px;
+    margin-top: 17px;
+}
+
+
+.blue-button {
+  width: 20px;
+  height: 20px;
+  background-color: #0050FF;
+  margin-top: 20px;
+  margin-left: 14px;
+}
+
+  .grey-box{
+    background-color:#E6DACE;
+    display:block;
+    width: 50%;
+    height: 45em;
+    margin-top: 35px;
+    
+}
+h1{
+position: relative;
+
+}
+
+
+.card{
+    grid-area: left;
+    display: flex;
+    flex-direction: column;
+    background-color: #F4ECE6;
+    right:-20em;
+    /* position: absolute; */
+    width: 22em;
+    right: 80;
+    bottom: 0;
+    left: 0;
+    align-items: center;
+    margin-top: 60px;
+    /* height: 30em; */
+    /* margin-left: 10%; */
+    box-shadow: grey -3px 3px 8px;
+    }
+   
+img{
+    width: 120px;
+    height: 120px;
+    border-radius: 50%; 
```

Comment:
nice circle!

Another great way to do this is with `clip-path`. Have a play https://bennettfeely.com/clippy/